pkg/virtctl/create/vm/vm.go

```diff
@@ -247,6 +248,11 @@ var volumeImportOptions = map[string]volumeImportFn{
 	snapshot: withVolumeSourceSnapshot,
 }
 
+var volumeImportSizeOptional = map[string]bool{
+	pvc:      true,
+	snapshot: true,
+}
```

Comment:
nit: I'd prefer
```suggestion
var volumeImportSizeMandatory = map[string]bool{
	pvc:      false,
	snapshot: false,
}
```
and save a **not** operation below.
But maybe it is my brain that starts burning when it see more than one negation :)